drivers/usbdev/composite.h

```diff
@@ -50,6 +50,13 @@
 #define COMPOSITE_CONFIGIDNONE        (0)  /* Config ID = 0 means to return to address mode */
 #define COMPOSITE_CONFIGID            (1)  /* The only supported configuration ID */
 
+/* Descriptor strings */
+
+#define COMPOSITE_MANUFACTURERSTRID     (1)
```

Comment:
these string is only used in drivers/usbdev/composite_desc.c, so don't need expose to public interface

Reply:
I know, it is only use COMPOSITE_SERIALSTRID in composite.c actually, but they are the same string ID group defs, so I moved them all.

Reply:
let's algin with other.lines.

Reply:
here

https://github.com/apache/nuttx/blob/dfcf2dee293a7ac97d1a151ca82f3c802a7650ee/drivers/usbdev/composite.c#L393